.github/workflows/release.yml

```diff
@@ -0,0 +1,96 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - 'master'
+
+jobs:
+
+  check:
+    if: ${{ github.repository_owner == 'MiczFlor' }}
+    runs-on: ubuntu-latest
+
+    outputs:
+      tag_name: ${{ steps.vars.outputs.tag_name }}
+      release_type: ${{ steps.vars.outputs.release_type }}
+      check_abort: ${{ steps.vars.outputs.check_abort }}
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set Output vars
+        id: vars
+        env:
+          BRANCH_NAME: ${{ github.ref_name }}
+        run: |
+          # Official SemVer Regex definition
+          # https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string
+          # Needed changes to the regex:
+          # - '?:' capture command needed to be removed as it wasn't working in shell
+          # - '\d' had to be replaced with [0-9]
+          #
+          # Release versions like 1.0.0, 3.5.0, 100.4.50000+metadata
+          REGEX_VERSION_RELEASE="^(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)(\+([0-9a-zA-Z-]+(\.[0-9a-zA-Z-]+)*))?$"
+          #
+          # Prerelease versions like 1.0.0-alpha, 3.5.0-whatsoever.12, 100.4.50000-identifier.12+metadata
```

Comment:
If the workflow runs only on master, do we still need _prerelease_?

Reply:
Not really needed. I just removed the stuff from future3 that a not working here, and left the rest identical.
But can remove it aswell, so there is no confusion.